pkg/cmd/condition/list_test.go

```diff
@@ -0,0 +1,129 @@
+// Copyright © 2019 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package condition
+
+import (
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/jonboulle/clockwork"
+	"github.com/tektoncd/cli/pkg/test"
+	cb "github.com/tektoncd/cli/pkg/test/builder"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	pipelinetest "github.com/tektoncd/pipeline/test"
+	tb "github.com/tektoncd/pipeline/test/builder"
+)
+
+func TestConditionList(t *testing.T) {
+	clock := clockwork.NewFakeClock()
+
+	seeds := make([]pipelinetest.Clients, 0)
+	conditions := []*v1alpha1.Condition{
+		tb.Condition("condition1", "ns", cb.ConditionCreationTime(clock.Now().Add(-1*time.Minute))),
+		tb.Condition("condition2", "ns", cb.ConditionCreationTime(clock.Now().Add(-20*time.Second))),
+		tb.Condition("condition3", "ns", cb.ConditionCreationTime(clock.Now().Add(-512*time.Hour))),
+	}
+	s, _ := test.SeedTestData(t, pipelinetest.Data{Conditions: conditions})
+	seeds = append(seeds, s)
+
+	testParams := []struct {
+		name      string
+		command   []string
+		input     pipelinetest.Clients
+		wantError bool
+		want      []string
+	}{
+		{
+			name:      "Found no conditions",
+			command:   []string{"ls", "-n", "notexist"},
+			input:     seeds[0],
+			wantError: false,
+			want:      []string{"No conditions found", ""},
+		},
+		{
+			name:      "Found conditions",
+			command:   []string{"ls", "-n", "ns"},
+			input:     seeds[0],
+			wantError: false,
+			want: []string{
+				"NAME         AGE",
+				"condition1   1 minute ago",
+				"condition2   20 seconds ago",
+				"condition3   3 weeks ago",
+				"",
+			},
```

Comment:
Can we have a test for https://github.com/tektoncd/cli/pull/319/files#diff-3b9bcd9007c070bb31b5ef44695b2016R81 also? 
Use PrependReactor to simulate error.

Reply:
@khrm Thank you for your review!!
I just added test case, please check this out.

Reply:
@khrm I think test error occured from outputting error message to Err stream.
In my opinion To fix this, the error message should be returned to caller not immediately output Err stream like below.
```
return fmt.Errorf("failed to list conditions from %s namespace \n%s", p.Namespace(), err)
```

But this is different from other error messages.
ex. https://github.com/tektoncd/cli/blob/master/pkg/cmd/pipeline/list.go#L93

If adding this test case I think we should create other pull request to fix all other files, how do you think 🤔??

Reply:
@khrm 
> @16yuki0702 Run hack/update-codegen.sh from root directory to fix the error.

Oh I totally misunderstood :sweat:
very sorry and thank you for your advise :smiley: